pkg/services/accesscontrol/models.go

```diff
@@ -193,7 +201,7 @@ type BuiltinRole struct {
 	Created time.Time
 }
 
-// Permission is the model for access control permissions.
```

Comment:
Fun fact: If you include a fullstop at the end of the comment, it will use it as the `title` of the generated model. If you don't it'll use it as `description`

This is the difference between the type being eventually generated as `PermissionIsTheModelForAccessControlPermissions` vs `Permission` 🫠